tools/validators/instance_validator/validate/instance_parser.py

```diff
@@ -42,34 +54,19 @@
                            syaml.Map({
                                'type': syaml.Str(),
                                'id': syaml.Str(),
-                               syaml.Optional('connections'):
-                                 syaml.MapPattern(syaml.Str(),
-                                                  syaml.Str()) |
-                                 syaml.Seq(
-                                     syaml.MapPattern(syaml.Str(),
-                                                      syaml.Str())),
+                               syaml.Optional('connections'): syaml.MapPattern(
+                                   syaml.Str(), syaml.Str()) | syaml.Seq(
+                                       syaml.MapPattern(syaml.Str(),
```

Comment:
and here.  And Why the | ?  I don't know what the other syntax option is.

Reply:
added a todo b/166472270 to revisit with the connections. Ray was trying things out, dont remember why we have a Seq here. We need to clean it up with the connections work